### DISCUSSION TITLE
Using the highlightDataSeries for a custom made legend

### DISCUSSION THREAD

**Discussion by pedromtsilva** [apexcharts/apexcharts.js#4444] [Q&A]
Hello guys. I am working with ApexCharts integration for Angular 13 and i'm in the process of creating a chart. However, I am making a component that will be switching between a line and a bar chart and i was wondering if i could get the highlightDataSeries propriety to bind it to my custom legend, that could control the data from both graphs. Is this possible?

**Comment by fctdev2024** [comment]
useless post any question here.
no good developer, only hostile people in here.
better get rid and move to another repo.
